rawkv/rawkv.go

```diff
@@ -652,10 +655,16 @@ func (c *Client) sendDeleteRangeReq(ctx context.Context, startKey []byte, endKey
 	}
 }
 
-func (c *Client) sendBatchPut(bo *retry.Backoffer, keys, values [][]byte) error {
+func (c *Client) sendBatchPut(bo *retry.Backoffer, keys, values [][]byte, ttls []uint64) error {
 	keyToValue := make(map[string][]byte, len(keys))
+	keyTottl := make(map[string]uint64, len(keys))
 	for i, key := range keys {
 		keyToValue[string(key)] = values[i]
+		if nil != ttls {
+			keyTottl[string(key)] = ttls[i]
+		} else {
+			keyTottl[string(key)] = 0
+		}
```

Comment:
```suggestion
		if len(ttls) > 0 {
			keyTottl[string(key)] = ttls[i]
		}
```
1. `len(tts) > 0` should be more easy to read.
2. you don't need to set map value to the default value.